lib/src/forms/multi_step_form.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:automated_testing_framework/automated_testing_framework.dart';
+import 'package:automated_testing_framework_plugin_flow_control/automated_testing_framework_plugin_flow_control.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:json_class/json_class.dart';
+import 'package:static_translations/static_translations.dart';
+
+class MultiStepForm extends TestStepForm {
+  const MultiStepForm();
+
+  @override
+  bool get supportsMinified => true;
+
+  @override
+  TranslationEntry get title =>
+      TestFlowControlTranslations.atf_flow_title_multi_step;
+
+  @override
+  Widget buildForm(
+    BuildContext context,
+    Map<String, dynamic> values, {
+    bool minify = false,
+  }) {
+    /// If this line is removed, [values] will still have a reference to the
+    /// original [steps] List, so modifying that here will modify it in the
+    /// [TestStep] without confirmation.
+    values['steps'] = List.from(values['steps'] ?? []);
+    return Column(
+      children: [
+        if (minify != true)
+          buildHelpSection(
+            context,
+            TestFlowControlTranslations.atf_flow_help_multi_step,
+            minify: minify,
+          ),
+        buildValuesSection(
+          context,
+          [
+            buildEditText(
+              context: context,
+              id: 'name',
+              label: TestFlowControlTranslations.atf_flow_form_multi_step_name,
+              values: values,
+            ),
+            _StepsEditor(
+              values: values,
+            )
+          ],
+        ),
+      ],
+    );
+  }
+}
+
+class _StepsEditor extends StatefulWidget {
+  _StepsEditor({@required this.values});
+
+  final Map<String, dynamic> values;
+
+  @override
+  _StepsEditorState createState() => _StepsEditorState();
+}
+
+class _StepsEditorState extends State<_StepsEditor> {
+  ScrollController _scrollController;
+  List<TestStep> _steps;
+  Translator _translator;
+
+  @override
+  void initState() {
+    super.initState();
+    _scrollController = ScrollController();
+    _steps = JsonClass.fromDynamicList(
+          widget.values['steps'],
+          (map) => TestStep.fromDynamic(map),
+        ) ??
+        [];
+    _translator = Translator.of(context);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Container(
+          height: 300,
```

Comment:
This is minor, but please add the `.0` to all decimals.  That guarantees the compiler knows the right value to pick.  It's not important here per se, but following that convention in all the code ensures that when you want a double, the line:
```
var expectedDouble = 0.0;
```
... is actually a double.  Where as...
```
var notDouble = 0;
```
... is an int.